.eslint-dictionary.json

```diff
@@ -1,4 +1,5 @@
 [
+  "75pk49boud2olipfda0ke3snic",
```

Comment:
Instead of adding this to the dictionary, just add the eslint-disable comment in line where it appears.